### PR TITLE
Update stork scheduler for 1.22+

### PR DESCRIPTION
**Pull request by kvaps**
upstream https://github.com/libopenstorage/stork/pull/982

### CODE REVIEW

**Review by WanzenBug — APPROVED** [review]
I trust that you tested this? TBH I don't really feel like testing stork again :sweat_smile:

File: charts/piraeus/templates/stork-deployment.yaml [hunk 1]
```diff
@@ -11,22 +11,27 @@ data:
...
-          "filterVerb": "filter",
-          "prioritizeVerb": "prioritize",
-          "weight": 5,
-          "enableHttps": false,
-          "nodeCacheCapable": false
-        }
-      ]
-    }
+  stork-config.yaml: |-
+    apiVersion: kubescheduler.config.k8s.io/v1beta3
```
Comment (kvaps) [inline charts/piraeus/templates/stork-deployment.yaml:15]: Yeah, small note, `kubescheduler.config.k8s.io/v1beta3` is working with 1.23+, while `kubescheduler.config.k8s.io/v1beta2` is working with 1.22+.

I think we can use `semverCompare` method here, just say me which versions we want to support?

**Comment by kvaps** [comment]
And may be it makes more sense to drop STORK in favor of linstor-scheduler-extender, so it would be nice to see it in official piraeus repository https://github.com/piraeusdatastore/piraeus-operator/issues/270

File: charts/piraeus/templates/stork-deployment.yaml (hunk 1 above)
Comment (WanzenBug) [reply charts/piraeus/templates/stork-deployment.yaml:15]: General guide is, we want to support every "stable" release, i.e. those that are listed here: https://www.downloadkubernetes.com/

I'm guessing we can't easily support 1.20 and 1.21?

**Review by WanzenBug — APPROVED** [review]

**Pull request merged**